backend/src/feed/connector.rs

```diff
@@ -83,20 +83,21 @@ impl FeedConnector {
                     hash => self.chain_hash = hash,
                 }
 
-                self.aggregator.send(Subscribe {
-                    chain: payload.into(),
-                    feed: ctx.address(),
-                })
-                .into_actor(self)
-                .then(|res, actor, _| {
-                    match res {
-                        Ok(true) => (),
-                        // Chain not found, reset hash
-                        _ => actor.chain_hash = 0,
-                    }
-                    async {}.into_actor(actor)
-                })
-                .wait(ctx);
+                self.aggregator
+                    .send(Subscribe {
+                        chain: payload.into(),
+                        feed: ctx.address(),
+                    })
+                    .into_actor(self)
+                    .then(|res, actor, _| {
+                        match res {
+                            Ok(true) => (),
+                            // Chain not found, reset hash
+                            _ => actor.chain_hash = 0,
+                        }
+                        async {}.into_actor(actor)
+                    })
+                    .wait(ctx);
             }
             "send-finality" => {
                 self.aggregator.do_send(SendFinality {
```

Comment:
Question unrelated to this PR (and really about line 115 below): why are we doing our own ping/pong thing rather than relying on the builtin ping pong mechanism?

Reply:
I've put that in to measure client latency, though that's not displayed anywhere other than via `console.log` in the FE currently.

Reply:
You mean like `their_pong_to_our_ping.timestamp - our_ping.timestamp`?

I wonder if surfacing that info in the lower websocket layer would be useful to have. Is it true that most websocket impls respond to `PING`/`PONG` frames right away (soketto does this)? If so it'd be a bit more accurate than doing it ourselves.

Maybe open a `nicetohave` ticket to investigate?